internal/bguploader_test.go

```diff
@@ -176,8 +176,8 @@ func addTestDataFile(t *testing.T, dir string, i int) {
 	}
 }
 
-func testFilename(i int) string {
-	return fmt.Sprintf("B%010d", i)
+func testFilename(i string) string {
+	return fmt.Sprintf("%08d%08d%08s", 1, 1, i)
```

Comment:
Will the new functionality work OK when wal-push is invoked for not WAL-formatted name?
PostgreSQL archived 00000007.history for example.
Or from time to time users archive their stuff :)

Reply:
It picks up any file generated in the directory and creates the metadata file. 
For example
0000000200000007000000C7.partial.json
0000000200000007000000C7.partial.lz4
00000003.history.json
00000003.history.lz4
0000000200000007000000C4.00000028.backup.json
0000000200000007000000C4.00000028.backup.lz4